### PR TITLE
Added Fuse runtime with circuit breaker mission implemented.

### PR DESCRIPTION
**Pull request by valdar**

### CODE REVIEW

**Comment by quintesse** [comment]
LGTM. Btw, you can remove the `booster.yaml` files from the booster repositories. They are not used anymore.

**Comment by kcbabo** [comment]
Hey @quintesse :  which BU are you waiting on for confirmation?

**Comment by kcbabo** [comment]
FWIW, I'm the Fuse PM and I'm good with adding this mission.

File: circuit-breaker/fuse/common.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+name: Spring Boot Circuit Breaker Example
```
Comment (Ladicek) [inline circuit-breaker/fuse/common.yaml]: Is there a risk of confusion with RHOAR Spring Boot?

**Comment by gastaldi** [comment]
We'll merge when the "DON'T MERGE YET" title is removed from this PR

**Comment by kcbabo** [comment]
I noticed the Node runtime entries have a Technical Preview banner on them.  We should have the same for Fuse right now as Fuse 7.0 is still in tech preview.

**Comment by quintesse** [comment]
@gastaldi I actually added the "DONT MERGE YET" to the title to make sure that nobody would merge it before this was properly discussed.

**Comment by quintesse** [comment]
@kcbabo AFAIK the tech preview banner is part of the Node icon itself (inside the `metadata.json`)

File: circuit-breaker/fuse/common.yaml (hunk 1 above)
Comment (ALRubinger) [reply circuit-breaker/fuse/common.yaml]: Yes.

**Comment by ALRubinger** [comment]
Fuse team, we're thrilled to have you on board here!  Hoping to get you lined in shortly and encourage your implementing the other missions as well!

@kcbabo is authoritative as stakeholder from the middleware BU.

However we also need @Ladicek signoff that the Booster is working:

1) Addition of the new runtime does not break any existing UI in the Launcher
2) As contracted in all target OSO environments.

**Comment by valdar** [comment]
@Ladicek changed the name to avoid confusion with RHOAR.
@quintesse I think @kcbabo was referring to the one in the homepage, which is part of the frontend not of the booster-catalog if I remember correctly?

**Comment by valdar** [comment]
@quintesse tested locally with master backend and frontend working both with firefox and chrome; if you point me to the version you are using for backend and frontend I can see if I can help with the icon problem.

**Comment by quintesse** [comment]
@valdar I just checked and it's not part of the icon actually. Both on the homepage and on the Runtime selection page. I'm guessing that's a change that will have to be made in the frontend. Right @edewit ?

The version I'm using is just the latest from master in both cases.

**Comment by gastaldi** [comment]
I confirm that I can see the logo.

**Comment by Ladicek** [comment]
Now hold on a second. I want to clarify one thing: I _don't_ feel responsible for testing non-RHOAR boosters. I don't want _anyone_ outside of RHOAR to expect RHOAR QE to cover their boosters. I hope I'm stating the obvious here, but I feel I have to point it out explicitly.

**Comment by aileenc** [comment]
Fuse QE (mmelko) will test the Fuse boosters.

**Comment by Ladicek** [comment]
At the same time, I'd also like to say that I'm happy to discuss how we test the RHOAR boosters with anyone who needs to test their boosters. Our CI setup is fully opensource: https://github.com/rhoar-ci

**Comment by benoitf** [comment]
hello, where can I find properties file commands to use for this project ? as for example maven commands.

something like for spring-boot circuit breaker
https://github.com/fabric8-launcher/launcher-documentation/blob/master/docs/topics/readme/cd-circuit-breaker-spring-boot.properties#L1-L2

There are commands to run but they're in README.adoc so more difficult to parse

**Pull request merged**